internal/services/privatednsresolver/private_dns_resolver_inbound_endpoint_resource.go

```diff
@@ -80,7 +81,7 @@ func (r PrivateDNSResolverInboundEndpointResource) Arguments() map[string]*plugi
 					"private_ip_address": {
```

Comment:
Can we leave a comment above this so we know this has been reviewed at
```suggestion
					// NOTE: O+C can remain here since the API will return a value for this if `private_ip_allocation_method` is `Dynamic` and this cannot be changed afterwards i.e. there is no scenario where this would need to be unset 
					"private_ip_address": {
```